### DISCUSSION TITLE
Unable to see the 'Create New Collaboration Session' Option in VsCode

### DISCUSSION THREAD

**Discussion by medalidridi** [eclipse-oct/open-collaboration-tools#58] [Q&A] [answered]
I am unable to see the 'Create New Collaboration Session' quick pick in the VS Code extension. Does anyone know why this option might not be appearing? Additionally, if I have deployed my own instance, is it acceptable to enter the server IP address in the extension settings and not a URL (formatted as 123.456.78.9:8100)? Thank you.

![image](https://github.com/user-attachments/assets/4b67ea95-0847-44d4-ac68-d036b2d44754)

**Comment by msujew** [comment, marked as answer]
Hey @medalidridi,

Do you have a workspace open? If there is no workspace, there's nothing available to share.

And yes, it's should be fine to simply use an IP for the address of the server instance.
